packages/stripe_checkout/test/stripe_checkout_test.dart

```diff
@@ -0,0 +1,5 @@
+
+
+void main() {
+  
```

Comment:
I think we can make some meaningful tests (e.g. that the redirect is called)

Reply:
I don't think this can be done with unit/widget testing as kWeb is constant and WebView are platform views. We should do some integration testing but it is not straightforward right now